test/filters/range_test.rb

```diff
@@ -1,44 +1,59 @@
-require 'test_helper'
-require 'elasticquery/filters/range'
+require "test_helper"
+require "elasticquery/filters/range"
 
 class RangeFilter < MiniTest::Test
   def setup
-    @filter = Elasticquery::Filters::Range.new 'year', lte: 2015, gte: 1990
-    @lte_filter = Elasticquery::Filters::Range.new 'age', lte: 58
-    @gte_filter = Elasticquery::Filters::Range.new 'age', gte: 18
+    @filter = Elasticquery::Filters::Range.new "year", lte: 2015, gte: 1990
+    @lte_filter = Elasticquery::Filters::Range.new "age", lte: 58
+    @gte_filter = Elasticquery::Filters::Range.new "age", gte: 18
   end
 
   def test_filter_valid_if_has_at_least_lte_or_gte_option
-    filter = Elasticquery::Filters::Range.new 'year', lte: 1
+    filter = Elasticquery::Filters::Range.new "year", lte: 1
     assert filter.valid?
-    filter = Elasticquery::Filters::Range.new 'year', gte: 0
+    filter = Elasticquery::Filters::Range.new "year", gte: 0
     assert filter.valid?
   end
 
   def test_filter_invalid_if_has_no_options
-    filter = Elasticquery::Filters::Range.new 'year'
+    filter = Elasticquery::Filters::Range.new "year"
     refute filter.valid?
   end
 
   def test_to_hash_should_return_query
     filter = Elasticquery::Filters::Range.new :year, lte: 1, gte: 0
-    assert_equal({query: {filtered: {filter: {and: {range: {year: {lte: 1, gte: 0}}}}}}}, filter.to_hash)
+    assert_equal({query: {filtered: {filter: {and: [{range: {year: {lte: 1, gte: 0}}}]}}}}, filter.to_hash)
   end
 
   def test_to_hash_with_one_param
     filter = Elasticquery::Filters::Range.new :year, lte: 1
-    assert_equal({query: {filtered: {filter: {and: {range: {year: {lte: 1}}}}}}}, filter.to_hash)
+    assert_equal({query: {filtered: {filter: {and: [{range: {year: {lte: 1}}}]}}}}, filter.to_hash)
+  end
+
+  def test_to_hash_support_cache_option
+    filter = Elasticquery::Filters::Range.new :year, lte: 1, _cache: true
+    assert_equal({query: {filtered: {filter: {and: [{range: {year: {_cache: true, lte: 1}}}]}}}}, filter.to_hash)
+  end
+
+  def test_to_hash_support_execution_option
+    filter = Elasticquery::Filters::Range.new :year, lte: 1, gte: 0, execution: "fielddata"
+    assert_equal({query: {filtered: {filter: {and: [{range: {year: {lte: 1, gte: 0, execution: "fielddata"}}}]}}}}, filter.to_hash)
```

Comment:
Line is too long. [131/80]<br>Space inside { missing.<br>Space inside } missing.